### PR TITLE
ADM-357 create a new project

### PR DESCRIPTION
**Pull request by fengxinhou**
## Summary

create a new project

## Before

null

## After

Click create a new project button go to this page

Stepper under home title

Back in step 2 and 3 can go back to step 1 and 2

Next in step 1 and 2 can go to step 2 and 3

### CODE REVIEW

File: frontend/src/main.tsx [hunk 1]
```diff
@@ -3,11 +3,15 @@ import ReactDOM from 'react-dom/client'
...
 import { store } from './store/store'
+import { ThemeProvider } from '@mui/material'
+import theme from '@src/theme'
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
-    <Provider store={store}>
-      <App />
-    </Provider>
+    <ThemeProvider theme={theme}>
```
Comment (guzhongren) [inline frontend/src/main.tsx:11]: any usage for ThemeProvider here?

**Review by devonzhang-tw — APPROVED** [review]

File: frontend/__tests__/src/components/metrics/MetricsStepper/MetricsStepper.test.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      expect(getByText(label)).toBeInTheDocument()
+    })
+    expect(getByText(NEXT)).toBeInTheDocument()
+    expect(getByText(BACK)).toBeInTheDocument()
+  })
+
+  it('should show metrics config step when click back button given config step ', () => {
+    const { getByText } = setup()
+
+    fireEvent.click(getByText(BACK))
```
[the remark this answers is not in the document]
Comment (LanxinWang) [reply frontend/__tests__/src/components/metrics/MetricsStepper/MetricsStepper.test.tsx:46]: done

File: frontend/src/main.tsx (hunk 1 above)
Comment (LanxinWang) [reply frontend/src/main.tsx:11]: change theme color

File: frontend/__tests__/src/components/metrics/MetricsStepper/MetricsStepper.test.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (LanxinWang) [reply frontend/__tests__/src/components/metrics/MetricsStepper/MetricsStepper.test.tsx:46]: done

File: frontend/__tests__/src/features/stepperSlice.test.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import stepperReducer, { nextStep, backStep } from '@src/features/stepper/StepperSlice'
+
+describe('counter reducer', () => {
+  it('should get 0 when handle initial state', () => {
+    expect(stepperReducer(undefined, { type: 'unknown' })).toEqual({
```
[the remark this answers is not in the document]
Comment (LanxinWang) [reply frontend/__tests__/src/features/stepperSlice.test.tsx:6]: extract done

File: frontend/e2e/src/pages/metrics.test.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+test('should render metrics page', async ({ page }) => {
+  await page.goto('/index.html')
+
+  await page.getByRole('button', { name: 'Create a new project' }).click()
+
+  steps.map(async (label) => {
+    await expect(page.getByText(label)).toBeVisible()
+  })
+  await expect(page.getByText('Step 1')).toBeVisible()
+  await expect(page.getByText('Next')).toBeVisible()
```
[the remark this answers is not in the document]
Comment (LanxinWang) [reply frontend/e2e/src/pages/metrics.test.ts]: done

File: frontend/e2e/src/pages/metrics.test.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  await expect(page.getByText('Back')).toBeVisible()
+
+  await page.getByRole('button', { name: 'Next' }).click()
+  await expect(page.getByText('Step 2')).toBeVisible()
```
[the remark this answers is not in the document]
Comment (LanxinWang) [reply frontend/e2e/src/pages/metrics.test.ts:27]: done

**Pull request merged**